app/uk/gov/hmrc/exports/controllers/DeclarationController.scala

```diff
@@ -72,13 +72,20 @@ class DeclarationController @Inject()(
       }
     }
 
-  def findAll(status: Option[String], pagination: Page): Action[AnyContent] =
+  def findAll(
+    status: Option[String],
+    pagination: Page,
+    sortBy: Option[String],
```

Comment:
Any chance we can wrap these in a `Sort` class a bit like `Page`?

Reply:
It would mean on other controllers that might use this we can reuse all the parsing/validation code

Reply:
I see you've got a `DeclarationSort` class lower down. You could use `Binder` e.g in `Page` to have that as a method parameter

Reply:
Done